kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverReportExtension.kt

```diff
@@ -221,6 +221,91 @@ public interface KoverReportsConfig {
      * ```
      */
     public fun verify(config: Action<KoverVerifyReportConfig>)
+
+    /**
+     * Configure coverage printing to the log for current report variant.
+     * ```
+     *  log {
+     *      onCheck = true
+     *
+     *      filters {
+     *          // ...
+     *      }
+     *      header = null
+     *      format = "<entity> line coverage: <value>%"
+     *      groupBy = GroupingEntityType.APPLICATION
+     *      coverageUnits = MetricType.LINE
+     *      aggregationForGroup = AggregationType.COVERED_PERCENTAGE
+     *  }
+     * ```
+     */
+    public fun log(config: Action<KoverLogReportConfig>)
+}
+
+/**
+ * Configuration of coverage printing to the log task for current report variant.
+ * ```
+ *  log {
+ *      onCheck = true
+ *
+ *      filters {
+ *          // ...
+ *      }
+ *      header = null
+ *      format = "<entity> line coverage: <value>%"
+ *      groupBy = GroupingEntityType.APPLICATION
+ *      coverageUnits = MetricType.LINE
+ *      aggregationForGroup = AggregationType.COVERED_PERCENTAGE
+ *  }
+ * ```
+ */
+public interface KoverLogReportConfig {
+    /**
+     * Override common filters only for logging report.
+     */
+    public fun filters(config: Action<KoverReportFilters>)
+
+    /**
+     * Print coverage when running the `check` task.
+     * `null` by default, for Kotlin JVM and Kotlin MPP triggered on `check` task, but not for Android.
+     */
+    public var onCheck: Boolean
+
+    /**
+     * Add a header line to be output before the lines with coverage.
+     *
+     * By default, the header is missing.
+     */
+    public var header: String?
+
+    /**
+     * Format of the string to output coverage for the specified in [groupBy] group.
+     *
+     * Placeholders used:
+     *  - `<value>` - coverage value
+     *  - `<entity>` - name of the entity by which the grouping took place. `application` if [groupBy] is [GroupingEntityType.APPLICATION].
+     *
+     * Default value is `<entity> line coverage: <value>%`.
+     */
+    public var format: String?
+
+    /**
+     * Specifies by which entity the code for separate coverage evaluation will be grouped.
+     * Default is [GroupingEntityType.APPLICATION]
+     */
+    public var groupBy: GroupingEntityType?
+
+    /**
+     * Specifies which metric is used for code coverage verification.
+     * Default is [MetricType.LINE]
+     */
+    public var coverageUnits: MetricType?
+
+    /**
+     * Specifies type of lines counter value to compare with minimal and maximal values if them defined.
```

Comment:
Is this value really 'compared'? Or just 'printed'?
Also, not clear what 'if them defined' means. 'If they are defined'?